src/tools/rest-frontend/resources/structure.json

```diff
@@ -0,0 +1,1896 @@
+[
+    {
+        "name": "Documentation",
+        "type": "submenu",
+        "ref": "documentation",
+        "children": [
+            {
+                "name": "Getting started",
+                "type": "listfiles",
+                "ref": "docgettingstarted",
+                "children": [
+                    {
+                        "name": "Why Elektra?",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/WHY.md"
+                        },
+                        "slug": "why-elektra"
```

Comment:
I think you are looking at the wrong file - this is the output file that is generated. The slug is used in the URL.

We could remove the file from the repository, to be honest. The structure.json.in is sufficient to reconstruct it (as long as the repo doesn't change).